### PR TITLE
Add support for --cluster-name or --cluster-id to databricks cluster commands

### PR DESCRIPTION
**Pull request by areese**
…commands

Add information about what is being removed to the databricks groups remove command
Update the license in setup.py
Add support for --cluster-name or --cluster-id to databricks libraries commands

### CODE REVIEW

**Comment by andrewmchen** [comment]
Since cluster names are not unique in the Databricks backend, I'm not sure it makes sense to have a `--cluster-name` argument for the `databricks clusters get` command. That would suggest that the name is unique...

**Comment by areese** [comment]
@andrewmchen it actually returns all of the clusters when you have more than one with the same name.
We have a large number of clusters, and right now, we have to look up the cluster id first, then run the get command again with the id we got from the name.

The --cluster-name is a shortcut, and returns the data for all of the clusters with that name.

**Comment by areese** [comment]
@andrewmchen as discussed, I'll refactor this to raise an error when there is more than one cluster-id with the same name.

**Comment by andrewmchen** [comment]
Great, happy to review once you've pushed your changes.

**Comment by areese** [comment]
@andrewmchen I've updated the commands to fail if there is more than one cluster with the same name.
I've also. gone through and updated the tests.

**Review by andrewmchen — COMMENTED** [review]
I think this PR is almost ready to go. I made a small edit which removed code from `sdk/service.py`. We shouldn't edit that file since the code is autogenerated. One last concern I have is what `databricks clusters get --cluster-name` returns. I think it's a bit weird that it returns the cluster-id instead of the cluster spec. Thoughts?

File: databricks_cli/clusters/cli.py [hunk 1]
```diff
@@ -154,17 +156,23 @@ def delete_cli(api_client, cluster_id):
...
-def get_cli(api_client, cluster_id):
+def get_cli(api_client, cluster_id, cluster_name):
     """
     Retrieves metadata about a cluster.
     """
-    click.echo(pretty_format(ClusterApi(api_client).get_cluster(cluster_id)))
+    if cluster_id is not None:
+        click.echo(pretty_format(ClusterApi(api_client).get_cluster(cluster_id)))
+    else:
+        clusters_by_name = ClusterApi(api_client).get_cluster_id_for_name(cluster_name)
```
[the remark this answers is not in the document]
Comment (areese) [reply databricks_cli/clusters/cli.py:174]: You're correct, it should return the cluster spec either way, it shouldn't get returning just the cluster id.
I think there was a new cluster id command that got dropped.

**Comment by areese** [comment]
@andrewmchen how is service.py generated?

File: databricks_cli/clusters/cli.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (areese) [reply databricks_cli/clusters/cli.py]: This is resolved.  :)

**Comment by andrewmchen** [comment]
`service.py` is generated by some code currently only living in Databrick's private repo unfortunately.

**Comment by areese** [comment]
@andrewmchen ty.

**Comment by areese** [comment]
@andrewmchen actually, I think I can revert the changes to sdk/service.py, I think they were added for lint

**Comment by areese** [comment]
@andrewmchen fixed, I reverted the sdk/service.py changes and tox passes for me, let's see what Travis thinks.

**Review by andrewmchen — APPROVED** [review]

**Comment by areese** [comment]
Thank you.

**Pull request merged**